ApiBundle/Transformer/WidgetCollectionTransformer.php

```diff
@@ -21,7 +24,10 @@ public function transform($widgetCollection)
         $facade = new WidgetCollectionFacade();
 
         foreach ($widgetCollection as $widget) {
-            $facade->addWidget($this->getTransformer('widget')->transform($widget));
+            $role = $this->getWidgetRole($widget);
+            if ($this->authorizationChecker->isGranted($role)) {
+                $facade->addWidget($this->getTransformer('widget')->transform($widget));
+            }
```

Comment:
Are you sure of this functionnality ? I thought the widgets were always displayed but there contents could be adjusted dependeing on the user.